CONTRIBUTORS.md

```diff
@@ -1598,7 +1598,7 @@ The Symfony Connect username in parenthesis allows to get more information
  - Viacheslav Sychov
  - Helmut Hummel (helhum)
  - Matt Brunt
- - Jack Thomas
+ - Jack Thomas (jackthomasatl)
```

Comment:
this file is autogenerated, so you must not edit it

Reply:
Got it. How does my username get attached? I signed up for symfony community with my email address... Or what do I need to do to link my username?

Reply:
Change reverted